### PR TITLE
Add SetTextSize viewer function for adjusting HUD text size

### PR DESCRIPTION
**Pull request by Hsifnus**
This PR adds a `SetTextSize` viewer function accessible in Python for changing the HUD text size, which before was fixed at a font size of 18 units for `qtosgrave` and 10 units for `qtcoinrave`.

### `qtosgrave` screenshots

In this screenshot, the text size is the default value of 18 units
Screenshot from 2024-03-13 13-26-02

Here, the text size is adjusted to 12 units
Screenshot from 2024-03-13 13-34-19

Here, the text size is adjusted to 24 units
Screenshot from 2024-03-13 13-34-40

### `qtcoinrave` screenshots

In this screenshot, the text size is the default value of 10 units
Screenshot from 2024-03-14 09-43-27

Here, the text size is adjusted to 14 units
Screenshot from 2024-03-14 09-44-00

Here, the text size is adjusted to 18 units
Screenshot from 2024-03-14 09-45-03

Here, the text size is adjusted to 26 units
Screenshot from 2024-03-14 09-45-32

### CODE REVIEW

File: plugins/qtcoinrave/qtcoinviewer.h [hunk 1]
```diff
@@ -412,7 +417,9 @@ public slots:
     std::list< boost::shared_ptr<IvDragger> > _plistdraggers;     /// draggers drawn
     SoEventCallback* _eventKeyboardCB;
 
+    SoFont* _messagefont;
```
Comment (braineo) [inline plugins/qtcoinrave/qtcoinviewer.h:420]:
```suggestion
    SoFont* _messageFont;
```

**Review by braineo — CHANGES_REQUESTED** [review]

File: plugins/qtcoinrave/qtcoinviewer.cpp [hunk 2]
```diff
@@ -640,6 +645,98 @@ void QtCoinViewer::SetUserText(const string& userText)
...
+    if (size < 18.0) {
+        return 14.0;
+    }
+    if (size < 26.0) {
+        return 18.0;
+    }
+    return 26.0;
+}
+
+SbVec3f QtCoinViewer::_GetMessageBaseTranslation()
```
Comment (braineo) [inline plugins/qtcoinrave/qtcoinviewer.cpp:702]: i think having some doc covering _GetMessageBaseTranslation and _GetMessageShadowTranslation would be good

Comment (Hsifnus) [reply plugins/qtcoinrave/qtcoinviewer.cpp:707]: added more doc on those methods in [9b1f9c6](https://github.com/rdiankov/openrave/pull/1366/commits/9b1f9c6bb1d9ac48235c30039e18bf5634ace930)

**Review by braineo — APPROVED** [review]
LGTM

**Comment by rdiankov** [comment]
thanks~

**Pull request merged**